### PR TITLE
feat: Clickable tags and labels on Single Article

### PR DESCRIPTION
**Pull request by abbyoung**
# SC-1400

Related PRs: 
* https://github.com/USSF-ORBIT/ussf-portal-cms/pull/316
* https://github.com/USSF-ORBIT/ussf-portal/pull/260

## Proposed changes

* Makes tags and labels clickable on Article page
* Tags and labels link to filtered search results page
* Adds searched query as input in search box on results page
* Updates `Internal News` category to `USSF News`
* Adds label and tag to Single Article in Storybook

## Setup

* In the CMS, create an article with tags and labels
* In the portal, view the article and confirm and clicking the tags and labels navigates to the search page with filtered results

### As the original developer, I have

- [x] Met the acceptance criteria, or will meet them in subsequent PRs or stories
  - ... 
- [x] Created new stories in Storybook if applicable
  - [ ] Checked that all Storybook accessibility checks are passing
- [x] Created/modified automated unit tests in Jest
  - [ ] Including jest-axe checks when UI changes
- [x] Created/modified automated E2E tests in Cypress
  - [ ] Including cypress-axe checks when UI changes
  - [ ] Checked that the E2E test build is not failing
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)
- [ ] Requested a design review for user-facing changes
- For any new migrations/schema changes:
  - [ ] Followed guidelines for zero-downtime deploys

### As code reviewer(s), I have

- [x] Pulled this branch locally and tested it
- [ ] Reviewed this code and left comments
- [x] Checked that all code is adequately covered by tests
  - [x] Checked that the E2E test build is not failing
- [ ] Made it clear which comments need to be addressed before this work is merged
- [x] Considered marking this as accepted even if there are small changes needed
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [ ] Checked responsiveness in mobile, tablet, and desktop
  - [ ] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [ ] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a designer reviewer, I have

- [x] Checked in the design translated visually
- [x] Checked behavior
- [x] Checked different states (empty, one, some, error)
- [x] Checked for landmarks, page heading structure, and links
- [x] Tried to break the intended flow
- Performed [a11y testing](https://github.com/trussworks/accessibility/blob/master/sample_a11y_testing_process.md):
  - [x] Checked responsiveness in mobile, tablet, and desktop
  - [x] Checked keyboard navigability
  - [ ] Tested with [VoiceOver](https://dequeuniversity.com/screenreaders/voiceover-keyboard-shortcuts) in Safari
  - [ ] Checked VO's [rotor menu](https://github.com/trussworks/accessibility/blob/master/README.md#how-to-use-the-rotor-menu) for landmarks, page heading structure and links
  - [x] Used a browser a11y tool to check for issues (WAVE, axe, ANDI or Accessibility addon tab for Storybook)

### As a test user, I have

- Run through the [Test Script](hhttps://docs.google.com/spreadsheets/d/1eV8UK0aJZ0qrzjnXf5SJ_uRiV7bpsj3yrhRFEkjOvV4/edit?usp=sharing):
  - [ ] On commercial internet in IE11
  - [ ] On commercial internet in Firefox
  - [ ] On commercial internet in Chrome
  - [ ] On commercial internet in Safari
  - [ ] On NIPR in IE11
  - [ ] On NIPR in Firefox
  - [ ] On NIPR in Chrome
  - [ ] On NIPR in Safari
  - [ ] On a mobile device in Firefox
  - [ ] On a mobile device in Chrome
  - [ ] On a mobile device in Safari
- [ ] Added any anomalous behavior to this PR

---

### CODE REVIEW

File: src/constants/index.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export const CONTENT_CATEGORIES = {
     value: 'InternalNews',
```
Comment (abbyoung) [inline src/constants/index.ts:31]: Waiting to remove these until they are fully deprecated

**Review by shkeating — COMMENTED** [review]
there is something funny going on with the heights and alignments of the different kinds of tags. I have the branch checked out and am looking into sorting it out.

image

**Comment by shkeating** [comment]
okay they should be sorted now. here's what I have on my end:
image

File: src/components/Search/Search.tsx [hunk 2]
```diff
@@ -49,6 +55,7 @@ const Search = ({ disabled }: { disabled?: boolean }) => {
             name="q"
             placeholder="What are you looking for today?"
             disabled={disabled}
+            defaultValue={query}
```
Comment (gidjin) [inline src/components/Search/Search.tsx:58]: Suggestion(non-blocking): can you add a test that checks the new query feature?

**Review by gidjin — APPROVED** [review]
LGTM!

**Review by shkeating — APPROVED** [review]

**Pull request merged**